src/config_parser.ml

```diff
@@ -440,7 +440,7 @@ let dhcp_option end_of_directive =
 
```

Comment:
in the above, from the dnsmasq manpage, the `option:` prefix is only used for named options, not for the integers:

> to set the default
              route option to 192.168.4.4, use `--dhcp-option=3,192.168.4.4` or
              `--dhcp-option=option:router,192.168.4.4` and to set the time-
              server address to 192.168.0.4, use `--dhcp-option=42,192.168.0.4`
              or `--dhcp-option=option:ntp-server,192.168.0.4`.  The special
              address `0.0.0.0` means "the address of the system running
              dnsmasq".

But we can do this in a separate PR.